packages/js/src/utils/config.ts

```diff
@@ -0,0 +1 @@
+export const PRODUCTION_BACKEND_URL = 'https://api.novu.co';
```

Comment:
We need a prop to override this in JS to support EU as well (this is not for this PR).

Reply:
We have a prop on the `Novu` class --> `options.backendUrl` and `options.socketUrl` that allows changing the backend services URLs, but maybe we can provide the bool prop that will allow to switch to EU?